marklogic-data-hub/src/test/resources/test-config/databases/staging-database.json

```diff
@@ -0,0 +1,5 @@
+{
+  "database-name": "data-hub-STAGING",
+  "range-element-index" : [],
```

Comment:
Are these empty arrays intentionally here to clear out the indexes?

Reply:
Yes. To restore the staging database index config to the previous state, just as for data-hub-FINAL database. Since there were no indexes previously, I have added empty arrays